dev-ops/README.md

```diff
@@ -98,6 +98,32 @@ source /mysql-dump/dump.sql;
 zcat /mysql-dump/dump_new.sql.gz | mysql -h localhost -u cbio -pP@ssword1 cbioportal
 ```
 
+### Refreshing cBioPortal
+
+**Any logged-in MedBook user may refresh cBioPortal.** Take this into consideration when granting access to MedBook, and double-check the validity of cBioPortal data if you suspect it may have been updated in error.
+
+To refresh cBioPortal with the latest list of collaborators in the WCDT collaboration, run the following in the Javascript console while logged into MedBook.
+```
+Meteor.call("refreshCBioPortalAccess");
+```
+
+To refresh the cBioPortal with the latest WCDT data, run the following in the Javascript console while logged into MedBook. This code will open a new tab to view the log file for the job. This tab will display a 404 error initially; once the job finishes, a log file will be available at the URL opened.
+
+```
+Meteor.call("refreshCBioPortalData", {
+  // replace these three IDs with the data you'd like to load
```

Comment:
I recall you saying that there were some issues testing because the chosen form didn't have the exact same samples as the sample group; is that correct? If so, make a note that each of the IDs must represent the exact same samples (no more or less)